modules/_labs/tabs/react/lib/TabPanels.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {TabPanelProps} from './TabPanel';
+
+type TabPanel = React.ReactElement<TabPanelProps>;
+
+export interface TabPanelsProps {
+  /**
+   * A list of TabPanel components.
+   */
+  children: TabPanel | TabPanel[];
+}
+
+const TabPanels = ({children}: TabPanelsProps) => {
+  const clonedChildren = React.Children.map(children, (child, index) => {
+    if (!React.isValidElement(child)) {
+      return child;
+    }
+    return React.cloneElement(child, {index});
```

Comment:
I added this so you don't need to manually pass `index`. It lessens performance, but it's nice for simple use cases. We should probably add a check here and just return the child if `child.props.index` is defined.

In the documentation for `TabPanel`, we can add a note saying performance for the tab will be better if you manually set the index